core-service/src/main/java/com/crmfoodestablishment/coreservice/mapper/DishMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.crmfoodestablishment.coreservice.mapper;
+
+import com.crmfoodestablishment.coreservice.dto.DishDto;
+import com.crmfoodestablishment.coreservice.entity.Dish;
+import com.crmfoodestablishment.coreservice.entity.Menu;
+import org.mapstruct.*;
+import java.util.List;
+
+@Mapper(
+        unmappedTargetPolicy = ReportingPolicy.IGNORE
+)
+public interface DishMapper {
+
+    List<DishDto> mapDishToDishDto(List<Dish> dish);
+
+    @Mapping(target = "menu", ignore = true)
```

Comment:
why is your solution so complecated here?)
You can write smth like this @Mapping(target = "menu", source = menu), since u don't process that object somehow after, you are only setting it